### PR TITLE
feature/165: Upgraded to virtual-schema-common-document 9.0.0

### LINKED ISSUE #165: Upgrade to virtual-schema-common-document 8.x
`virtual-schema-common-document` version 8.x contains breaking changes regarding the [Unified connection definition ](https://github.com/exasol/virtual-schema-common-document/releases/tag/8.0.0).

Upgrading to the latest version will result in a breaking change also for dynamodb-virtual-schema, see [s3-document-files-virtual-schema 2.0.0](https://github.com/exasol/s3-document-files-virtual-schema/releases/tag/2.0.0) as an example.

`com.exasol:virtual-schema-common-document 7.0.1 -> 8.1.1`

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #165

### CODE REVIEW

File: doc/user-guide/user_guide.md [hunk 1]
```diff
@@ -43,21 +43,41 @@ For AWS use:
...
-    USER 'fakeMyKeyId'
-    IDENTIFIED BY 'fakeSecretAccessKey';
-
+    TO ''
+    USER ''
+    IDENTIFIED BY '{
+        "awsAccessKeyId": "fakeMyKeyId", 
+        "awsSecretAccessKey": "fakeSecretAccessKey", 
+        "awsRegion": "eu-central-1",
+        "awsEndpointOverride": "localhost:8000'",
```
Comment (kaklakariada) [inline doc/user-guide/user_guide.md:78]:
```suggestion
        "awsEndpointOverride": "localhost:8000",
```

File: pom.xml [hunk 2]
```diff
@@ -1,41 +1,24 @@
...
         <junit.version>5.8.2</junit.version>
         <jacoco.version>0.8.8</jacoco.version>
-        <vs-common-document.version>7.0.1</vs-common-document.version>
+        <vs-common-document.version>9.0.0</vs-common-document.version>
         <gpg.skip>true</gpg.skip>
         <!--
                 M4 does not fail tests if beforeAll fails
                 M5 fails in integration tests with a strange error
                 -->
         <surefire.and.failsafe.plugin.version>3.0.0-M3</surefire.and.failsafe.plugin.version>
```
Comment (kaklakariada) [inline pom.xml:20]:
```suggestion
```
Defined in parent pom

File: pom.xml [hunk 3]
```diff
@@ -394,49 +185,19 @@
...
-                <version>3.2.2</version>
-                <executions>
-                    <execution>
-                        <id>default-jar</id>
-                        <phase>none</phase>
-                    </execution>
-                </executions>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <configuration>
+                    <forkCount>0</forkCount>
```
Comment (kaklakariada) [inline pom.xml:192]:
```suggestion
                    <useModulePath>false</useModulePath>
```
Try using this

**Review by kaklakariada — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (jakobbraun) [reply pom.xml:20]:
```suggestion
```

**Review by kaklakariada — APPROVED** [review]

**Pull request merged**